vars/stageScanForSnyk.groovy

```diff
@@ -0,0 +1,46 @@
+def call(def context, def snykAuthenticationCode, def buildFile, def organisation) {
+  if (!snykAuthenticationCode) {
+    println("Skipping Snyk Scan due to missing authentication code (to enable pass as parameter a snyk service account authentication code to this stage)")
+    return
+  }
+  if (!organisation) {
```

Comment:
`organisation ` is a mandatory param, but here it seems that we can infer a default. Should we make it optional then?

Reply:
I made it mandatory